sql/src/planner.rs

```diff
@@ -564,16 +564,26 @@ impl<'a, P: MetaProvider> PlannerDelegate<'a, P> {
     }
 
     fn drop_table_to_plan(&self, stmt: DropTable) -> Result<Plan> {
```

Comment:
Maybe we should add unit test for this function.

Reply:
`test_drop_statement_to_plan` already covers this `fn`.